src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -521,14 +515,34 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       await this.props.dataViews.clearCache();
     }
 
-    const deletes = selectedSavedObjects
-      .filter((object) => !object.meta.hiddenType)
-      .map((object) => savedObjectsClient.delete(object.type, object.id, { force: true }));
-    await Promise.all(deletes);
+    const deleteStatus = await bulkDeleteObjects(
+      http,
+      selectedSavedObjects.map(({ id, type }) => ({ id, type }))
```

Comment:
Isn't that going to be confusing for the user? The user selects those items and requests deletion. If we filter them out, they won't disappear, nor any error message is shown.

Reply:
ATM, that is what's happening in the UI anyway. As a demo, I select three saved objects, two of which are hidden (cases). The modal table filters out the cases SO's and these aren't deleted from the SO list view.
The network tab shows that we still include the hidden types in the bulk_delete payload, which returns errors for each hidden object.
![bulk_delete_containing_hidden_SO](https://user-images.githubusercontent.com/11909450/212984741-0edce8d8-24b4-4de7-9682-1e28eee61a38.gif)

My point is that we shouldn't even send these as part of the payload since they don't get deleted anyway.

Reply:
Thanks for the demo. I missed that part. It should be filtered out now 👍